modules/ROOT/pages/mule-4-api-javadoc.adoc

```diff
@@ -1,9 +1,20 @@
-= Mule API Javadoc
+= Mule Runtime Engine Javadoc
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: java, classes, javadoc, reference, objects, methods
 
-== Java Classes
+MuleSoft provides Javadocs for the Community Edition of the Mule Runtime Engine.
+The Javadoc contains definitions for classes, fields and their data types, and also describes methods including their usage, parameters and return values.
 
-* https://www.mulesoft.org/docs/site/4.1.1/apidocs/[Mule 4.1.1 API]
\ No newline at end of file
+Read the documentation for Mule SDK if you want an easier way to extend the Mule Runtime Engine to suit your needs.
+
+You can find links to the Javadocs below, ordered by release version:
```

Comment:
", ordered by release version" is not needed bc there's only one version at the moment.